### PR TITLE
Initializer postman sync

### PR DESCRIPTION
**Pull request by akiskips**
There were some descrepancies between the postman collection and the jsons used for the initializer. 
Updated both so that postman requests can be issued to the registries with sample data from the initializer

### CODE REVIEW

File: docs/postman/sample_requests/sample_interaction.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
           {
             "idType": "URI",
             "type": "GlobalReference",
-            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/",
+            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/sampleSM",
```
Comment (Manu3756) [inline docs/postman/sample_requests/sample_interaction.json:28]: remove /sampleSM

Comment (akiskips) [reply docs/postman/sample_requests/sample_interaction.json]: reverted it to the original value without /sampleSM

File: docs/postman/sample_requests/sample_interaction.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
           {
             "idType": "URI",
             "type": "GlobalReference",
-            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/",
+            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/10JF-1234-Jf14-PP22",
```
Comment (Manu3756) [inline docs/postman/sample_requests/sample_interaction.json:28]: just remove /10JF-1234-Jf14-PP22.

File: src/compose/volumes/initializer/populateStorageAdapterRegistry.json [hunk 3]
```diff
@@ -3,14 +3,14 @@
     "adapterid": "storage-adapter-mongodb",
     "url": "http://i40-aas-storage-adapter-mongodb:3100/submodels",
     "name": "mongo-adapter",
-    "submodelid": "sap.com/aas/submodels/part-100-device-information-model/sampleSM",
-    "submodelsemanticid": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/sampleSM"
+    "submodelid": "sap.com/aas/submodels/part-100-device-information-model/10JF-1234-Jf14-PP22",
+    "submodelsemanticid": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/10JF-1234-Jf14-PP22"
```
Comment (Manu3756) [inline src/compose/volumes/initializer/populateStorageAdapterRegistry.json:7]: Just remove /10JF-1234-Jf14-PP22. Only from semanticId not from id field.

File: src/compose/volumes/initializer/populateStorageAdapterRegistry.json [hunk 4]
```diff
@@ -3,14 +3,14 @@
...
   },
   {
     "adapterid": "storage-adapter-ain",
     "url": "http://i40-aas-storage-adapter-ain:80/submodels",
     "name": "ain-adapter-cloud",
-    "submodelid": "sap.com/aas/submodels/part-100-device-information-model/10JF-1234-Jf14-PP22",
+    "submodelid": "sap.com/aas/submodels/part-100-device-information-model/10JF-1234-Jf14-PP22-r",
```
Comment (Manu3756) [inline src/compose/volumes/initializer/populateStorageAdapterRegistry.json:13]: why added "-r"?

Comment (akiskips) [reply src/compose/volumes/initializer/populateStorageAdapterRegistry.json:13]: we have 3 different adapters to route to

File: src/compose/volumes/initializer/populateStorageAdapterRegistry.json (hunk 3 above)
Comment (akiskips) [reply src/compose/volumes/initializer/populateStorageAdapterRegistry.json]: ok updated

**Pull request merged**